pkg/jsonhttp/jsonhttptest/testing_mock_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package jsonhttptest_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+)
+
+// assert is a test helper that validates a functionality of another helper
+// function by mocking Errorf, Fatal and Helper methods on testing.TB.
```

Comment:
are you sure this should read `testing.TB`?

Reply:
I am using testing.TB here in assert and mock for consistency with the Request function which uses testing.TB interface in order to inject the mock. Type mock can use testing.T struct, as well. I can change it if you think that it would be better.